aggregation/src/idl/aggregations.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto2";
+option java_package = "fk.prof.aggregation.proto";
+option java_outer_classname = "AggregatedProfileModel";
+
+message ProfileSourceInfo {
+	optional string ip = 1;
+	optional string hostname = 2;
+	optional string processName = 3;
+	optional string zone = 4;
+	optional string instanceType = 5;
+}
+
+enum ProfileStatus {
+	Completed = 0;	// got all the samples
+	Partial = 1;	// got only few of the samples.
+	Retried = 2;	// may have duplicated samples because of retries 
+	Aborted = 3;    // {see if its required}.
+}
+
+message ProfileSummary {
+	optional string startTime = 1;				// HH:mm:ss
+	optional string endTime = 2;				// HH:mm:ss
+	optional uint32 recorderVersion = 3;
+	optional ProfileSourceInfo sourceInfo = 4;
+	optional ProfileStatus status = 5;
+}
+
+message ProfileList {
+	repeated ProfileSummary info = 1;
+}
+
+
+// Additional properties computed for the context
+message ContextProps {
+	optional uint32 sampleCount = 1;
```

Comment:
Will add that later. I couldn't finalize on its structure (as we discussed that coverage can vary from 1 recorder to another).